docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/managing-channels.md

```diff
@@ -1,6 +1,6 @@
 # Managing Channels
 
-This article describes how to manage Channels in Liferay Commerce. Channels have a 1:1 relationship with sites. To learn more about the using Channels in Liferay Commerce, see: [Introduction to Channels](./introduction-to-channels.md).
```

Comment:
"Channels have a 1:1 relationship with sites"

Seems like an odd place to put this info. Since it's talking about the characteristics of a channel and not how to manage it, it feels like it should exist in the intro. If not in the intro, somewhere which isn't the introductory paragraph.